.env.example

```
@@ -184,7 +186,27 @@ AURORA_CACHE_TOKEN_IN_REDIS=false
+# GitHub auth mode: app | oauth | hybrid (default: app)
+#   app    — GitHub App only (recommended, the only path most deployments need)
+#   oauth  — Legacy user OAuth only. For on-prem deployments that cannot host
+#            their own GitHub App. Requires GH_OAUTH_CLIENT_ID/SECRET below.
+#   hybrid — Both paths active. Frontend dialog shows both CTAs; auth router
+#            prefers App tokens and falls back to OAuth.
+GITHUB_AUTH_MODE=app
+
+# GitHub App. See server/connectors/github_connector/SETUP_GITHUB_APP.md or run
+# `python3 server/scripts/register_github_app.py --org <your-org>` to bootstrap.
+GITHUB_APP_ID=
+GITHUB_APP_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_APP_CLIENT_ID key should go before the GITHUB_APP_ID key

(UnorderedKey)